data/phi0588/abo001/phi0588.abo001.perseus-lat2.xml

```diff
@@ -73,6 +75,7 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?>
     </profileDesc>
     
     <revisionDesc>
+      <change when="2023-01-18" who="Alison Babeu">Updated header and added in page level link</change>
```

Comment:
A few minor header changes don't require the change log update. Feel free to skip this if you don't want the extra work.

Reply:
Okely dokely!